assets/src/sass/editor/_global.scss

```diff
@@ -3,101 +3,39 @@ body {
 	line-height: normal;
 }
 
-.block-editor-block-list__layout.is-root-container > .wp-block:not(.is-reusable),
-.is-reusable > .block-library-block__reusable-block-container > .block-editor-block-list__layout > .wp-block:not(.is-reusable):not([data-align="full"]) {
-	@include responsive-padding(0 strip-px($block-padding * 0.5), 0 strip-px($block-padding));
+.is-layout-constrained {
+	--wp--style--global--content-size: 980px;
+	--wp--style--global--wide-size: 1020px;
 }
-
-.wp-block.block-editor-default-block-appender {
-	max-width: $content-column-padded-rem;
+.post-type-post {
+	--wp--style--global--content-size: 40rem;
+	--wp--style--global--wide-size: 980px;
 }
 
-.block-editor-block-list__layout > .wp-block {
-	max-width: $content-column-padded-rem;
-
-	.has-background,
-	&.has-background {
-		padding: 1.25em 2.375em;
-	}
-
-	&.has-background {
-		max-width: $content-column-rem;
-	}
-
-	.single-post & {
-		max-width: $content-column-post-padded-rem;
-
-		&.has-background {
-			max-width: $content-column-post-rem;
-		}
-
-		&[data-align="wide"] {
-			max-width: $content-column-padded-rem;
-
-			.has-background {
-				padding: 1.25em 2.375em;
-			}
-		}
+.has-background > .wp-block-group__inner-container > :last-child {
+	margin-bottom: 0;
+}
```

Comment:
These editor styles remain a bit of a mess, but are simpler than before, and I'm hesitant to further simplify until we have a new target style to implement as part of the refresh project.